lib/netsuite/records/translation.rb

```diff
@@ -0,0 +1,13 @@
+module NetSuite
+  module Records
+    class Translation
+      include Support::Fields
+      include Support::Records
+      include Namespaces::ListAcct
+
+      def initialize(attributes = {})
+        initialize_from_dynamic_attributes_hash(attributes)
```

Comment:
@fabiensebban  Why did this need to use a new dynamic attributes approach? Are the fields included here dynamic?